File: deno_demo/entity/sysUser.ts
import { Model, DataTypes } from "../deps.ts";

class SysUserModel extends Model {
    static table = "sys_user";
    static fields = {
        asid: {
            primaryKey: true,
            autoIncrement: true,
            type: DataTypes.INTEGER
        },
        id: {
            type: DataTypes.STRING
        },
        userName: {
            type: DataTypes.STRING,
            field: "user_name"
        },
        password: {
            type: DataTypes.STRING
        },
        name: {
            type: DataTypes.STRING
        },
        idcardNo: {
            type: DataTypes.STRING,
            field: "idcard_no"
        },
        sex: {
            type: DataTypes.STRING
        },
        orgCode: {
            type: DataTypes.STRING,
            field: "org_code"
        },
        orgName: {
            type: DataTypes.STRING,
            field: "org_name"
        }
    };
}

export default SysUserModel;

File: com.aswqazx.webflux/src/main/java/com/aswqazx/webflux/entity/SysUser.java
package com.aswqazx.webflux.entity;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

/**
 * @author OMNIS
 */
@Data
@Table("sys_user")
public class SysUser {

    @Id
    private int asid;

    private String id;

    private String name;

    private String password;

    private String userName;

    private String idcardNo;

    private int sex;
}
